cmd/raid.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"github.com/bmc-toolbox/common"
+	"github.com/spf13/cobra"
+)
+
+var raidCmd = &cobra.Command{
+	Use:   "raid",
+	Short: "Configures various types of RAID",
+	Long:  "Configures various types of RAID",
+}
+
+func init() {
+	raidCmd.PersistentFlags().String("raid-type", common.SlugRAIDImplLinuxSoftware, "RAID Type (linuxsw,hardware)")
+	// markFlagAsRequired(raidCmd, "raid-type")
+
+	rootCmd.AddCommand(raidCmd)
+}
+
+func getRaidType(cmd *cobra.Command) string {
+	raidType := GetString(cmd, "raid-type")
+	if raidType == "" {
+		raidType = common.SlugRAIDImplLinuxSoftware
```

Comment:
Setting a default here may not be necessary since you've specified the default value in the PersistentFlag() command above?

Reply:
also noted we could get rid of getRaidObjectType entirely.